src/main/java/org/quantumbadger/redreader/viewholders/VH1TextIcon.java

```diff
@@ -0,0 +1,17 @@
+package org.quantumbadger.redreader.viewholders;
+
+import android.view.View;
+import android.widget.ImageView;
+
+import org.quantumbadger.redreader.R;
+
+public class VH1TextIcon extends VH1Text {
```

Comment:
Calling these `VH1TextIcon` and `VH2TextIcon` is a bit confusing -- a comment (and/or more descriptive names) would make things clearer.